source/app/models/user.rb

```diff
@@ -1,2 +1,19 @@
 class User < ActiveRecord::Base
+  enum role: {user: 0, admin: 100}
```

Comment:
This seemed a quick and dirty way to do it. I started doing a roles table, and entertained doing permissions, and users_permissions tables, but decided that was coding for the future, and there's no reason to infer more complex permissions.